### ISSUE TITLE
A11y wave

### ISSUE THREAD

**Issue by w3bdesign** [w3bdesign/carasent-react-typescript#86]
https://wave.webaim.org/report#/http://carasent-react-typescript.vercel.app/

**Comment by w3bdesign** [comment]
What It Means
A form control does not have a corresponding label.

Why It Matters
If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.

**Comment by w3bdesign** [comment]
Løst med https://github.com/w3bdesign/carasent-react-typescript/commit/bfce6fa118b34c9fd498ab43a1e7c7b9252f0b71
